geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/jts/util/util.scala

```diff
@@ -0,0 +1,10 @@
+package org.locationtech.geomesa.spark.jts.util
```

Comment:
this class is causing the license check to fail - @mojodna do you want copyright on these files? We can add a new header template with your copyright, otherwise we can add the standard CCRi one:
```
/***********************************************************************
 * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Apache License, Version 2.0
 * which accompanies this distribution and is available at
 * http://www.opensource.org/licenses/apache2.0.php.
 ***********************************************************************/
```

Reply:
CCRi is fine.

Reply:
thanks, I'll go ahead and add it